indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -74,33 +76,79 @@
+                    <span class="abstract-metadata-context">
+                        {{ render_track(review.track, classes='abstract-track-tag i-tag outline') }}
+                    </span>
                 </div>
-                <div class="abstract-comment-badges flex">
-                    <div class="stretch">
-                        <span>
+            </div>
+            <div class="i-timeline-item-box header-indicator-top">
+                <div class="i-box-header flex">
+                    <div class="abstract-comment-badges">
+                        {% set action = _render_review_state(review) %}
+                        {% set endaction = '</span>'|safe %}
+
+                        {% if review.proposed_action.name == 'accept' %}
+                            {% if review.proposed_contribution_type %}
+                                {% set contribution_type -%}
+                                    <strong>{{ review.proposed_contribution_type.name|escape }}</strong>
+                                {%- endset %}
+                                {% trans type=contribution_type|safe -%}
+                                    Proposed to {{ action }}Accept{{ endaction }} as {{ type }}
+                                {%- endtrans %}
+                            {% else %}
+                                {% trans action=_render_review_state(review) -%}
+                                    Proposed to {{ action }}Accept{{ endaction }}
+                                {%- endtrans %}
+                            {% endif %}
+                        {% elif review.proposed_action.name == 'reject' %}
                             {% trans action=_render_review_state(review) -%}
-                                Proposes to {{ action }}
+                                Proposed to {{ action }}Reject{{ endaction }}
+                            {%- endtrans %}
+                        {% elif review.proposed_action.name == 'change_tracks' %}
+                            {% trans type=contribution_type|safe -%}
+                                Proposed to {{ action }}Change Tracks{{ endaction }}
                             {%- endtrans %}
-                        </span>
+                        {% elif review.proposed_action.name in ('mark_as_duplicate', 'merge') %}
+                            {% set other_abstract = review.proposed_related_abstract %}
+                            {% set abstract_link -%}
+                                <a href="{{ url_for('.display_abstract', other_abstract) }}">
+                                    {{ other_abstract.title|escape }}
+                                </a>
+                            {%- endset %}
+                            {% set title = abstract_link|safe %}
+                            {% if review.proposed_action.name == 'merge' %}
+                                {% trans -%}
+                                    Proposed to {{ action }}Merge{{ endaction }} into {{ title }}
```

Comment:
seems strange for me to have it start with an uppercase letter (or does the rule where you can uppercase words in english e.g. in titles also apply for a case like this one)